src/Cli.Tests/InitTests.cs

```diff
@@ -164,8 +164,8 @@ bool expectSuccess
         /// </summary>
         [DataRow(null, "testcontainer", "", false, DisplayName = "Both database and schema are either null or empty.")]
         [DataRow("", "testcontainer", "testschema", false, DisplayName = "database is empty.")]
-        [DataRow("testDatabase", "testcontainer", "", false, DisplayName = "database is provided, Schema is null.")]
-        [DataRow("testDatabase", null, "", false, DisplayName = "database is provided, container and Schema is null/empty.")]
+        [DataRow("testDatabase", "testcontainer", "", true, DisplayName = "database is provided, Schema is null.")]
```

Comment:
Previously, we used to error out if schema file and database name both are not provided for cosmosDb_NoSQL.  I checked the logic in the ConfigGenerator.cs file in your PR. Shouldn't it still be false?